test/image/mocks/mapbox_scattercluster.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Let's improve this mock using the figure presented in this [demo](https://codepen.io/MojtabaSamimi/pen/xxOVLxK?editors=1000).
Thanks.